core/src/test/java/com/linecorp/armeria/client/retry/RetryingClientAuthorityHeaderTest.java

```diff
@@ -81,18 +77,8 @@ public void authorityIsDifferentByBackendsWhenRetry() {
         assertThat(res.contentUtf8()).contains("www.bar.com");
     }
 
-    @Test
-    public void authorityIsSameWhenSet() {
-        final HttpClient client = newHttpClientWithEndpointGroup();
-
-        final RequestHeaders headers = RequestHeaders.of(GET, "/",
-                                                         HttpHeaderNames.AUTHORITY, "www.armeria.com");
-        final AggregatedHttpResponse res = client.execute(headers).aggregate().join();
-        assertThat(res.contentUtf8()).contains("www.armeria.com");
```

Comment:
Again, this doesn't work anymore because Armeria will auto-set the `:authority` header. I think it's not a problem because a user can specify the same host name for the `Endpoint`s in his or her `EndpointGroup`.